### DISCUSSION TITLE
Regarding an issue of Technician form

### DISCUSSION THREAD

**Discussion by Bimbara28** [cepdnaclk/e19-co227-digital-reporting-of-technical-visits#80] [Q&A] [answered]
I just added some scss stylings to the Add New Technician form. But the Add mobile number and submit button are overlapping and they are at the top of the form instead of the bottom. Help me to solve this problem😢

**Comment by chandula00** [comment, marked as answer]
I went through ur SCSS code. there u had added some properties, Commonly for all buttons as 
```
button {
   position: absolute ;
   top: 5% ;
   right: 5%;
}
```

since it is common to all buttons, all of them are positioned as above. if u introduced a separate className for the close button, it may be helpful for u 😁
